content/doc/book/pipeline/development.adoc

```diff
@@ -0,0 +1,113 @@
+---
+layout: section
+title: Pipeline Development Tools
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= Pipeline Development Be
+
+In this section, we'll discuss tools and resources
+that help speed development of Jenkins Pipelines.
+
+== Creating
+
+The
+<<getting-started#built-in-documentation, built-in documentation>> and the
+<<getting-started#snippet-generator, Snippet Generator>>
+are good place to start when developing Jenkins Pipeline.
+
+=== Blue Ocean Editor (Declarative only)
+
+The
+<<../blueocean/pipeline-editor#, Blue Ocean Pipeline Editor>> provides a WYSIWYG
+way to create Delcarative Pipelines.
+
+=== Pipeline Unit Testing Framework
```

Comment:
I think these sub-sections should be ordered in the reverse order:

* Blue Ocean Pipeline Editor
* Declarative Linter
* Replay
* Pipeline Unit

I don't think Replay only lives in the realm of debugging, and since it's built in I think it should be further up.